src/App/App.js

```diff
@@ -1,27 +1,52 @@
 import React, { Component } from 'react';
 import Movies from '../Movies/Movies'
-import movieData from '../movieData'
 import NavBar from '../NavBar/NavBar'
 import MovieView from '../MovieView/MovieView'
+import {getAllMovies, getMovieByID, getMovieTrailerByID} from '../apiCalls.js'
 import '../App/App.scss';
 
 class App extends Component {
   constructor (){
     super () 
     this.state = {
-      movies: movieData.movies,
+      movies: [],
       toggled: false,
       currentMovie: {}
     } 
   }
 
-  toggleView = (movieID) => {
-    const matchedMovie = movieData.movies.find(movie => {
-      return movie.id === parseInt(movieID)
-    })
-       this.setState({
-        toggled: !this.state.toggled,
-        currentMovie: matchedMovie
+  componentDidMount() {
+    getAllMovies()
+      .then(data => this.setState({
+        movies: data.movies
+      }))
+      .catch(error => console.log(error))
```

Comment:
It's important to do something in your catch statement that will help your user. They are not likely to check the dev tools to see if there are any console errors. It would be better to set something in state, and then do a bit of conditional rendering if there is an error present.